main/webapp/modules/core/scripts/project/history-panel.js

```diff
@@ -303,7 +303,7 @@ HistoryPanel.prototype._showApplyOperationsDialog = function() {
         }
       };
       reader.addEventListener('error', function() {
-        alert('Error: Failed to read file');
+        alert($.i18n('Error: Failed to read file'));
```

Comment:
Not quite there yet. Can you look at other uses of `$.i18n` in the rest of the code base to understand how it is used? We also have some documentation about it: https://openrefine.org/docs/technical-reference/translating-ui

Reply:
@wetneb , it works fine now.